src/layout/layout.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './layout.css';
+import { UsersBar } from '../users_bar/users_bar';
+// import { get } from '../queries/fetcher';
```

Comment:
Please, delete comment. We have history from git.

Reply:
fixed